### PR TITLE
Bad internal link for bootnode docs

### LINKED ISSUE #415: Broken internal link on Create Private Network page
## Describe the bug

At the end of this page there is a link to step 3. The link doesn't work.
https://besu.hyperledger.org/en/stable/Tutorials/Private-Network/Create-Private-Network/#3-start-first-node-as-bootnode

link text:
To restart the private network in the future, start from 3. Start First Node as Bootnode.

- [ ] Missing content
- [x] Outdated content
- [ ] Wrong content
- [ ] Confusing or misleading content
- [ ] Other

## The broken page

https://besu.hyperledger.org/en/stable/Tutorials/Private-Network/Create-Private-Network

## Change suggestion

fix the link :) the link on the RHS works

### LINKED ISSUE #416: broken link on Configuring Privacy page
## Describe the bug

https://besu.hyperledger.org/en/stable/Tutorials/Privacy/Configuring-Privacy/

The "created the genesis file" step link doesn't work 
"This tutorial assumes you have completed setting up an IBFT 2.0 network to the point where you have created the genesis file." 

- [ ] Missing content
- [x] Outdated content
- [ ] Wrong content
- [ ] Confusing or misleading content
- [ ] Other

## The broken page

https://besu.hyperledger.org/en/stable/Tutorials/Privacy/Configuring-Privacy/

## Change suggestion

The genesis file is step 2 - but maybe linking to individual steps is a bad idea if they change.
https://besu.hyperledger.org/en/stable/Tutorials/Private-Network/Create-IBFT-Network/#2-create-a-configuration-file

### PR DESCRIPTION
**Pull request by ryjones**
fixes #415
fixes #416

### CODE REVIEW

File: docs/Tutorials/Private-Network/Create-Private-Network.md [hunk 1]
```diff
@@ -209,7 +209,7 @@ When finished using the private network, stop all nodes using ++ctrl+c++ in each
 !!!tip
 
     To restart the private network in the future, start from
-    [3. Start First Node as Bootnode](#3-start-first-node-as-bootnode).
+    [3. Start first node as a bootnode](#3-start-first-node-as-a-bootnode).
```
Comment (bgravenorst) [inline docs/Tutorials/Private-Network/Create-Private-Network.md:212]: This should be:

[3. Start the first node as a bootnode](#3-start-the-first-node-as-a-bootnode)

File: docs/Tutorials/Privacy/Configuring-Privacy.md [hunk 2]
```diff
@@ -8,8 +8,8 @@ Configuring a network that supports private transactions requires starting an Or
 Hyperledger Besu node. Besu command line options associate the Besu node with the Orion node.
 
 This tutorial assumes you have completed setting up an IBFT 2.0 network to the point where you have
-[created the genesis file](../Private-Network/Create-IBFT-Network.md#5-create-genesis-file). If
-not, complete steps 1 to 5 of the
+[created the genesis file](../Private-Network/Create-IBFT-Network.md#5-copy-the-node-private-keys-to-the-node-directories).
```
Comment (bgravenorst) [inline docs/Tutorials/Privacy/Configuring-Privacy.md:11]: Perhaps change this to:
`[created the genesis file and copied the private keys]`

**Comment by NicolasMassart** [comment]
I updated this PR description for the automatic linking between PR and issues to work as this is the reason why I did not notice this PR at first.

**Comment by ryjones** [comment]
I did not mean to cause duplicate work. Sorry, @NicolasMassart

**Comment by NicolasMassart** [comment]
> I did not mean to cause duplicate work. Sorry, @NicolasMassart

That's ok, don't worry, I'm just confused that I did not notice yours and thank you for contributing!

**Pull request merged**